### PR TITLE
Update the search doc

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4963

### CODE REVIEW

**Comment by dockerymick** [comment]
Troubleshoot update related to the cross-squad issue: https://issues.redhat.com/browse/ACM-4774

@SherinV will add a comment on the troubleshoot issue

**Review by SherinV — APPROVED** [review]

**Comment by SherinV** [comment]
Thanks, @dockerymick. /lgtm.

**Review by SherinV — APPROVED** [review]

**Comment by dockerymick** [comment]
@SherinV thanks Sherin for your feedback and approval! Pulling in another writer for peer review. This will be merged soon

File: troubleshooting/trouble_shared_memory.adoc [hunk 1]
```diff
@@ -54,3 +54,5 @@ oc delete pod search-postgres-xyz search-api-xzy
 ----
 oc get cm search-postgres -n ocm -o yaml
 ----
+
+See link:../observability/search_intro.adoc#search-customization[Search customization and configurations] for more information on adding environment variables.
```
Comment (swopebe) [inline troubleshooting/trouble_shared_memory.adoc:58]: this line grayed out for some reason.

**Review by swopebe — APPROVED** [review]
Looks so good!

File: troubleshooting/trouble_shared_memory.adoc (hunk 1 above)
Comment (dockerymick) [reply troubleshooting/trouble_shared_memory.adoc:58]: Good catch, I'm not sure why it is. Ran the link check tool as well 
Screenshot 2023-05-02 at 2 52 01 PM

**Comment by dockerymick** [comment]
Thanks for the reviews and approvals! Verified that the links are good. Merging this PR

**Pull request merged**